plugins/integrations/kubernetes-service/src/main/java/com/cloud/kubernetes/cluster/KubernetesClusterManagerImpl.java

```diff
@@ -1526,6 +1575,8 @@ public KubernetesClusterConfigResponse getKubernetesClusterConfig(GetKubernetesC
     }
 
     @Override
+    @ActionEvent(eventType = KubernetesClusterEventTypes.EVENT_KUBERNETES_CLUSTER_SCALE,
+            eventDescription = "scaling Kubernetes cluster", async = true)
     public boolean scaleKubernetesCluster(ScaleKubernetesClusterCmd cmd) throws CloudRuntimeException {
```

Comment:
```java
CallContext.current().setEventDetails("Current min size: ?? New min size: ??");
```
Can we also set event details here? It would be good to have extra information about the change itself in events.

Reply:
will check

Reply:
I tried this. There are too many variables to scale,
- size
- minSize
- maxSize
- offering
I've limited my change to add event details only when there is change in size for now